tools/footdown.py

```diff
@@ -62,7 +62,7 @@ def displacement(fk):
 def dataset(path, left_state, right_state, servos, fields, lr_duplicate, foot_delta, accelerometer, gryoscope):
     xs = []
     ys = []
-    for type_name, timestamp, msg in tqdm(nbs_decoder.decode(path), dynamic_ncols=True, unit="packet"):
+    for type_name, timestamp, msg, raw in tqdm(nbs_decoder.decode(path), dynamic_ncols=True, unit="packet"):
```

Comment:
what is `raw`? is it used anywhere?

Reply:
Raw is the raw bytes from the file. It's used in stats to get a size in bytes, and I figured it could be useful one day if you're ever just "passing through" protobufs and reencoding them. The other option would be to return the size instead of the raw bytes. and if you want to renencode you need to actually re serialise the protobufs